main.go

```diff
@@ -93,6 +94,7 @@ func main() {
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&webhookDisabled, "disable-webhooks", false, "Disable webhooks used to validate custom resources")
 	flag.StringVar(&webhookCertDir, "tls-cert-dir", "/etc/webhook/certs", "The directory with a tls.key and tls.crt for serving HTTPS requests")
+	flag.IntVar(&webhookServerPort, "webhook-server-port", 443, "The port that the webhook server serves at")
```

Comment:
We should allow overriding the value for `webhook-server-port` through Helm chart values similar how is done for `disable-webhooks`. Apparently the same option is missing for `tls-cert-dir` as well.

See: https://github.com/banzaicloud/koperator/blob/master/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L156

E.g.
```
{{- if .Values.webhook.enabled }}
  {{- if .Values.webhook.tls.certDir }}
  - --tls-cert-dir={{ .Values.webhook.tls.certDir }}      
  {{ end }}
  {{- if .Values.webhook.serverPort}}
  - --webhook-server-port={{ .Values.webhook.serverPort }}      
  {{ end }}
{{ else }}
   - --disable-webhooks
{{- end }}
```

Reply:
With the updated PR:
```
helm template charts/kafka-operator  | grep args -C 10
...
        - command:
            - /manager
          args:
            - --enable-leader-election
            - --cert-manager-namespace=cert-manager
            - --cert-manager-enabled=false
          image: "ghcr.io/banzaicloud/kafka-operator:v0.18.2"
          imagePullPolicy: IfNotPresent
          name: manager
          env:
            - name: POD_NAMESPACE
...
```
```
...
        - command:
            - /manager
          args:
            - --enable-leader-election
            - --cert-manager-namespace=cert-manager
            - --cert-manager-enabled=false
            - --tls-cert-dir=test/dir
            - --webhook-server-port=9443
          image: "ghcr.io/banzaicloud/kafka-operator:v0.18.2"
          imagePullPolicy: IfNotPresent
          name: manager
          env:
            - name: POD_NAMESPACE
...
```